src/app/layout.tsx

```diff
@@ -24,6 +21,10 @@ import { getColorScheme } from "~/lib/get-color-scheme"
 import { ColorSchemeInjector } from "./ColorSchemeInjector"
 import Providers, { mantineDefaultColorScheme, mantineTheme } from "./providers"
 
+import "@crossbell/connect-kit/colors.css"
+import "@mantine/core/styles.css"
+import "~/css/main.css"
+
```

Comment:
Using the previous plugin, we could not manually control the order of imports with side effects.